test/execution.jl

```diff
@@ -326,12 +326,12 @@ end
     @cuda kernel(arr)
     @test Array(arr)[] == 1
 
-    function kernel(ptr)
+    function kernel2(ptr)
```

Comment:
This is a broken change. The whole deal of this test is to ensure redefinitions are applied.

Reply:
done